lib/rucio/client/uploadclient.py

```diff
@@ -347,8 +348,12 @@ def _register_file(self, file, registered_dataset_dids):
         logger.debug('Registering file')
 
         # verification whether the scope exists
-        account_scopes = self.client.list_scopes_for_account(self.client.account)
-        if file['did_scope'] not in account_scopes:
+        account_scopes = []
+        try:
+            account_scopes = self.client.list_scopes_for_account(self.client.account)
+        except ScopeNotFound:
+            logger.warning('No socpe belonging to the account {}. Processing upload with given scope or no scope at all.'.format(self.client.account))
```

Comment:
In the `except` part we don't need the separate message, so a `pass` here is sufficient.